src/powfacpy/case_studies.py

```diff
@@ -340,4 +405,82 @@ def clear_parent_folders(self):
           parent_folder=parent_folder,
           error_if_non_existent=False,
           include_subfolders=True)  
+        
+  def _get_parameter_name_to_list_mapping(self):
+    """Returns a dictionary that maps the parameters to
+    a list including indexing.
+    Example:
+      If there are two parameters "p HV load" and "control 1",
+      the dictionary is
+      {"p HV load": "x[0], "control 1": x[1]}
+    """
+    par_name_to_list_mapping = {}
+    for par_num,par_name in enumerate(self.parameter_values.keys()):
+      par_name_to_list_mapping[par_name] = "x[" + str(par_num) + "]"  
+    return par_name_to_list_mapping
+
+  def export_results_of_study_cases(
+      self, 
+      export_dir=None, 
+      study_cases=None,
+      case_numbers=None,
+      results_obj="ElmRes",
+      results_variables_lists=None,
+      leave_csv_file_unchanged=False):
+    """Export the simulation results (ElmRes) of the study cases to csv files. 
+    The csv files are named according to the study case number (e.g. case0.csv, case1.csv,..)
+    Returns the full paths of the csv files.
+
+    Arguments:
+      - export_dir: directory for export (default is working directory)
+      - study_cases: study case objects
+      - case_numbers: corresponing study case numbers
+      - results_obj: string that used in GetFromStudyCase to get the ElmRes
```

Comment:
not quite clear to me what is meant here. 
would this just be "ElmRes" as in app.GetFromStudyCase("ElmRes") ?

Reply:
yes, I clarified that